File: scaffold/src/main/java/top/xuqingquan/web/QuickCallJs.java
package top.xuqingquan.web;

import com.tencent.smtt.sdk.ValueCallback;

public interface QuickCallJs {

    void quickCallJs(String method, ValueCallback<String> callback, String... params);

    void quickCallJs(String method, String... params);

    void quickCallJs(String method);


}

File: scaffold_web/src/main/java/top/xuqingquan/web/publics/QuickCallJs.kt
package top.xuqingquan.web.publics

import android.webkit.ValueCallback
import com.tencent.smtt.sdk.ValueCallback as X5ValueCallback

interface QuickCallJs {

    fun quickCallJs(method: String?, callback: ValueCallback<String>?, vararg params: String?)

    fun quickCallJs(method: String?, callback: X5ValueCallback<String>?, vararg params: String?)

    fun quickCallJs(method: String?, vararg params: String?)

    fun quickCallJs(method: String?)
}
